src/__tests__/getMonthName.test.ts

```diff
@@ -0,0 +1,43 @@
+import { it, expect, vi, beforeAll, afterAll, describe } from 'vitest';
+import getMonthName from '../utils/getMonthName';
+
+beforeAll(() => {
+  vi.useFakeTimers().setSystemTime(new Date('2001-04-04'));
+});
+afterAll(() => {
+  vi.useRealTimers();
+});
+
+it('should return current month name info when has no arguments passed', () => {
+  expect('Apr').toEqual(getMonthName());
```

Comment:
```suggestion
  expect(getMonthName()).toEqual('Apr');
```

Also for remaining tests in this file